nginx-ssl.conf

```diff
@@ -20,6 +20,8 @@ http {
             rewrite ^/seattle-2016/?.*$ /berlin-2016/ permanent;
             # generic code of conduct, newsletter redirects to berlin
             rewrite ^/(code-of-conduct|newsletter)(/)?(index.html)?$ /berlin-2016/ permanent;
+            # schedule, local-guide, cfp redirects to landing page
+            rewrite ^/.*(schedule|local-guide|cfp|cfp-thanks)(/)?(index.html)?$ /berlin-2016/ permanent;
```

Comment:
I cannot get my local docker instance to recognize these redirects, so I can't test the changes. 

I ran `make build-deploy-image` and then `make serve-https`; when I load https://localhost/seattle-2016 (or any other redirect from a non-existent page), I get a 404. @jgmize, do these new redirects look OK to you? There's one in each nginx file.